client/models.js

```diff
@@ -1,4 +1,6 @@
-import { has, assert, maybeHas, NONE_CATEGORY_ID, stringToColor } from './helpers';
+import { has, assert, maybeHas, NONE_CATEGORY_ID, stringToColor,
+         formatCurrency, getCurrencySymbol, currencyExists } from './helpers';
+import { store } from './store';
```

Comment:
OK, thatś what I feared. But then, it's hard to do something 'clean' to set a default value for the currency.